wavelink/player.py

```diff
@@ -302,6 +372,58 @@ async def _search(query: str | None) -> T_a:
         random.shuffle(self.auto_queue._queue)
         logger.debug(f'Player "{self.guild.id}" added "{added}" tracks to the auto_queue via AutoPlay.')
 
+        # Probably don't need this here as it's likely to be cancelled instantly...
+        self._inactivity_start()
+
+    @property
+    def inactive_timeout(self) -> int | None:
+        """A property which returns the time as an ``int`` of seconds to wait before this player dispatches the
+        :func:`on_wavelink_inactive_player` event.
+
+        This property could return ``None`` if no time has been set.
+
+        An inactive player is a player that has not been playing anything for the specified amount of seconds.
+
+        - Pausing the player while a song is playing will not activate this countdown.
+        - The countdown starts when a track ends and cancels when a track starts.
+        - The countdown will not trigger until a song is played for the first time or this property is reset.
+        - The default countdown for all players is set on :class:`~wavelink.Node`.
+
+        This property can be set with a valid ``int`` of seconds to wait before dispatching the
+        :func:`on_wavelink_inactive_player` event or ``None`` to remove the timeout.
+
+
+        .. wanring::
+
+            Setting this to a value of ``0`` or below is the equivalent of setting this property to ``None``.
+
+
+        When this property is set, the timeout will reset, and all previously waiting countdowns are cancelled.
+
+        - See: :class:`~wavelink.Node`
+        - See: :func:`on_wavelink_inactive_player`
+
+
+        .. versionadded:: 3.2.0
+        """
+        return self._inactivity_wait
+
+    @inactive_timeout.setter
+    def inactive_timeout(self, value: int | None) -> None:
+        if not value or value <= 0:
+            self._inactivity_wait = None
+            self._inactivity_cancel()
+            return
+
+        if value and value < 10:
```

Comment:
`if value` is redundant here as the initial `if not value` means there will always be a value at this point.
This can be just `if value <10:`

Reply:
👍